webapp/app/templates/react_template.py

```diff
@@ -0,0 +1,10 @@
+from flask import render_template
+
+REACT_COMPONENT = 'react_component.html'
```

Comment:
Why do we still need that constant? Couldn't we put that directly in line 6?

Reply:
Done!